### DISCUSSION TITLE
Git author must have access to the project on Vercel

### DISCUSSION THREAD

**Discussion by trym-slalom** [vercel/community#731] [Help] [answered]
Hi, I was just given admin access to a repo and am trying to deploy it on Vercel, but on BitBucket it says that the build failed and that "Git author must have access to the project on Vercel". Is there a way to fix this, or is it possible something that just takes a certain amount of time before it syncs up my admin access with Vercel? Thanks :))
 
<img width="600" alt="Screen Shot 2022-07-18 at 09 57 48" src="https://user-images.githubusercontent.com/107150415/179527757-825e4266-84d2-4e77-999a-05a53f2dbc99.png">

**Comment by amyegan** [comment, marked as answer]
Hi @trym-slalom. This error seems related to your Vercel account access rather than needing admin access to a Git repo. It looks like you are not recognized as a member of that Team in Vercel. I have a few questions to help us figure out what is going wrong here.

- Are you trying to deploy a Project on your own Vercel account, or are you a member of the Team that owns the Project? 
- Is your BitBucket account linked to your Vercel account?
- Is your [Git Metadata correctly configured](https://vercel.com/support/articles/why-aren-t-commits-triggering-deployments-on-vercel#correctly-configured-metadata)?

**Reply by trym-slalom** [reply]
Got it solved now. Apparently I was not part of the project on Vercel. One of my team-members invited me, and now everything is good whenever I deploy! Thanks for the helpful advice :))
